promscale/visualize-data/index.md

```diff
@@ -0,0 +1,11 @@
+# Visualize data in Promscale
+Promscale can be used to visualize the data in tools like
+[Grafana][grafana-homepage] and [Jaeger][jaeger-homepage]
+
+*   **Grafana**: [use Grafana to visualize metrics and traces][grafana-promscale].
+*   **Jaeger**: [use Jaeger Query to visualize traces][jaeger-promscale].
```

Comment:
I would not mention Jaeger Query here but just Jaeger. We can mention Jaeger Query in the details of the documentation.